### ISSUE TITLE
Youtube在cookie协议退出

### ISSUE THREAD

**Issue by livingfree2023** [1234567Yang/cf-proxy-ex#10]
无论是接受还是拒绝，都跳回引导页了

**Comment by 1234567Yang** [comment]
history的问题应该是，可能我现在比较忙以后解决

**Comment by qingyuan0o0** [comment]
用了昨天更新的[版本](https://git.doop.workers.dev/https://github.com)除了首页,任何页面都打不开了,好奇怪

**Comment by 1234567Yang** [comment]
> 用了昨天更新的[版本](https://git.doop.workers.dev/https://github.com)除了首页,任何页面都打不开了,好奇怪 返回的结果是 `Something is wrong while trying to get your cookie: siteCookie: null lastSite: undefined`

这是最新的测试网址：pull-test-pxy.wvusd.homes
经过测试是可以打开的，请您确保是最新版本？再复制粘贴一份试试？
* 最新版本Google map有BUG没法加载，所以我的示例网址还是旧版，正在试图修复

**Comment by qingyuan0o0** [comment]
> > 用了昨天更新的[版本](https://git.doop.workers.dev/https://github.com)除了首页,任何页面都打不开了,好奇怪 返回的结果是 `Something is wrong while trying to get your cookie: siteCookie: null lastSite: undefined`
> 
> 这是最新版本的测试网址：pull-test-pxy.wvusd.homes 经过测试是可以打开的，请您确保是最新版本？再复制粘贴一份试试？
> 
> * 最新版本Google map有BUG没法加载，所以我的示例网址还是旧版，正在试图修复
是的,确定是最新版的,因为只有最新版在首页有输入地址框部分[版本首页](https://git.doop.workers.dev/)

**Comment by 1234567Yang** [comment]
> 是的,确定是最新版的,因为只有最新版在首页有输入地址框部分[版本首页](https://git.doop.workers.dev/)

再粘贴一遍试试看。

**Comment by qingyuan0o0** [comment]
经过测试,失败的原因找到了,我原来做了一个worker是直接反向github.com的,资源就是git.xxx.worker.dev/name/项目这么用的,我今天直接把这个项目的代码换成你的worker代码,所以失败了,我删除掉,新建了,就正常了,真神奇...
